app/models/post.rb

```diff
@@ -0,0 +1,15 @@
+class Post < ActiveRecord::Base
+  belongs_to :user
+  has_many :likes
+  has_many :comments
+
+  def five_last_comments
+    comments.order('created_at').limit(5)
```

Comment:
- [ ] It should return the 5 most recent comments for a given post in descending order.

`comments.order('created_at Desc').limit(5)`